router/broker.go

```diff
@@ -98,6 +131,30 @@ func (b *broker) role() wamp.Dict {
 	return brokerRole
 }
 
+// PreInitEventHistoryTopics initializes storage for event history enabled topics.
+func (b *broker) PreInitEventHistoryTopics(evntCfgs []*TopicEventHistoryConfig) error {
+	for _, topicCfg := range evntCfgs {
+		if !topicCfg.Topic.ValidURI(b.strictURI, topicCfg.MatchPolicy) {
+			return fmt.Errorf("invalid topic/match configuration found: %s %s", topicCfg.Topic, topicCfg.MatchPolicy)
+		}
+
+		if topicCfg.Limit <= 0 {
+			return fmt.Errorf("invalid limit for topic configuration found: %d %s", topicCfg.Limit, topicCfg.Topic)
```

Comment:
This error message is complex without saying what is really wrong:

Why not just `errors.New("limit cannot be negative")` wrapped in `topicConfigError{}` that provides `topicCfg.Topic` if that's useful?